### PR TITLE
add *.out in .gitignore

### PR DESCRIPTION
**Pull request by jwpark-sungshin**
exclude *.out in .gitignore

resolved: #3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 *.obj
+*.out
```
Comment (pjbear1215) [inline .gitignore:2]: please consider the following
```suggestion
*.obj
*.out
*.o
*.exe
```

**Review by pjbear1215 — CHANGES_REQUESTED** [review]
This is looking great. I've left a few comments that should be addressed before it gets merged.

File: .gitignore (hunk 1 above)
Comment (jwpark-sungshin) [reply .gitignore:2]: OK. I'll do it.

**Review by pjbear1215 — APPROVED** [review]
Well done.

**Pull request merged**